helm/managed/logging/elastic/values.yaml

```diff
@@ -10,6 +10,13 @@ elasticsearch:
   minimumMasterNodes: 1 #default is 2, for minimal systems 1 master node should be adequate
   labels: {"managedby" : "HPCC"}
   clusterHealthCheckParams: "local=true" #local node health status
+  #clusterHealthCheckParams: "wait_for_status=yellow&timeout=50s"
+  readinessProbe:
```

Comment:
Why are we setting these values?

Reply:
When you wait "green" or "yellow" user may need tweak these parameters such as "delay" or "timeout"

Reply:
But we're not waiting on green/yellow, are we?
Does your change work w/ out these values?